frontend/src/components/auth/Login.tsx

```diff
@@ -48,6 +48,20 @@ const Login = () => {
     setSignup(true);
   };
 
+  type GoogleErrorResponse = {
+    error: string;
+    details: string;
+  };
+
+  const onFailure = (response: GoogleErrorResponse) => {
+    // https://stackoverflow.com/questions/63631849/google-sign-in-not-working-in-incognito-mode
+    if (response.error === "idpiframe_initialization_failed") {
+      console.warn("Google SignIn does not work on incognito mode.");
```

Comment:
do we want to show this error to users so they know why they could not log in? (unless it is already, i haven't been able to replicate this error locally)